src/convertToObject.js

```diff
@@ -3,16 +3,29 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = {};
+  const propertyValuePairs = sourceString.split(';');
+
+  for (const pair of propertyValuePairs) {
```

Comment:
**[IMPORTANT]**

[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. 

This for ... of cycle is ALSO meant to be an iteration. Please find a way to remove it and incorporate an iteration in its place.